CNV -30825: add KubevirtHCOSingleStackIPv6 (#62709)
openshift/openshift-docs: modules/virt-runbook-kubevirthcosinglestackipv6.adoc

// Do not edit this module. It is generated with a script.
// Do not reuse this module. The anchor IDs do not contain a context statement.
// Module included in the following assemblies:
//
// * virt/monitoring/virt-runbooks.adoc

:_content-type: REFERENCE
[id="virt-runbook-KubevirtHcoSingleStackIPv6"]
= KubevirtHcoSingleStackIPv6

[discrete]

> [id="meaning-kubevirthcosinglestackipv6"]

Reviewer: Is an id actually required in this case, since the title is discrete? 
> Same comment applies to all the other titles that are tagged as discrete

== Meaning

This alert fires when you install {VirtProductName} on a single stack
IPv6 cluster.

[discrete]
[id="impact-kubevirthcosinglestackipv6"]
== Impact

You cannot create virtual machines.

[discrete]
[id="diagnosis-kubevirthcosinglestackipv6"]
== Diagnosis

> * Check the cluster network configuration by running the following command:

Reviewer: ```suggestion
* Run the following command to check the cluster network configuration:
```
